### PR TITLE
logging_update_2

### PR DESCRIPTION
**Pull request by rflorenc**
removed logger tag output: https://bugzilla.redhat.com/show_bug.cgi?id=1400631
simplified installation using a deployer helper
added all ES _cat api endpoints
updated ansible logging_deployer.yaml path
updated readme
removed index saturation scripts

### CODE REVIEW

**Comment by rflorenc** [comment]
@wabouhamad 
@jeremyeder 

Part of the handover to SVT. Please review.
It was already demoed to / tested by @wabouhamad.

File: logging_metrics_performance/enterprise_logging/README.md [hunk 1]
```diff
@@ -19,16 +19,14 @@ Clone this repository.
...
-    
-    MASTER_URL="https://ip-xxx-xx-xx-xxx.us-xxxx-x.compute.internal:8443"
-    PUBLIC_MASTER_URL="https://ec2-xx-xxx-xxx-xxx.us-xxxx-x.compute.amazonaws.com:8443"
-
-    $ ./enterprise_logging_setup.sh ${MASTER_URL} ${PUBLIC_MASTER_URL} 
+    $ cat deployer.conf 
+    VERSION=v1.4
+    CLUSTER_SIZE=3
+    KIBANA_HOSTNAME=kibana.example.com
+    IMAGE_PREFIX=registry.ops.openshift.com/openshift3/
```
Comment (jeremyeder) [inline logging_metrics_performance/enterprise_logging/README.md]: This hostname needs to be abstracted out somehow.

File: logging_metrics_performance/enterprise_logging/deployer.conf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+VERSION=v1.4
+CLUSTER_SIZE=3
+KIBANA_HOSTNAME=kibana.example.com
+IMAGE_PREFIX=registry.ops.openshift.com/openshift3/
```
Comment (jeremyeder) [inline logging_metrics_performance/enterprise_logging/deployer.conf]: same here please.

**Comment by jeremyeder** [comment]
You guys rock!

**Pull request merged**